rosetta/docs/PGLE.md

```diff
@@ -61,12 +61,10 @@ PGLE found latency for async op custom-call-start.1 and (assumed)custom-call-don
 In order to get the best performance with PGLE, here is a list of all recommended XLA flags:
 ```
 export XLA_FLAGS="--xla_gpu_enable_latency_hiding_scheduler=true
---xla_gpu_enable_triton_gemm=false
 --xla_gpu_graph_level=0
---xla_gpu_enable_highest_priority_async_stream=true
 --xla_gpu_all_reduce_combine_threshold_bytes=1073741824
 --xla_gpu_all_gather_combine_threshold_bytes=1073741824
---xla_gpu_reduce_scatter_combine_threshold_bytes=1073741824
+--xla_gpu_reduce_scatter_combine_threshold_bytes=134217728
```

Comment:
Why change this number in a PR that remove flags?

Reply:
will revert to the previous, thanks